meta-refkit/classes/refkit-image.bbclass

```diff
@@ -230,7 +230,7 @@ NOHDD = "1"
 # Image creation: add here the desired value for the PARTUUID of
 # the rootfs. WARNING: any change to this value will trigger a
 # rebuild (and re-sign, if enabled) of the combo EFI application.
-REMOVABLE_MEDIA_ROOTFS_PARTUUID_VALUE = "12345678-9abc-def0-0fed-cba987654321"
+REMOVABLE_MEDIA_ROOTFS_PARTUUID_VALUE = "12345678-9abc-def0-0fed-cba987654329"
```

Comment:
And if we want to use this way of testing image I could just remove sd cards from Minnowboard and nuke Joules internal emmc so there will be no conflicts. Or we can just change the uuid.